lib/rules/indent.js

```diff
@@ -651,7 +651,7 @@ module.exports = {
             if (elements.length > 0 && node.type === "ArrayExpression") {
                 elements.forEach(el => {
                     if (el.loc.start.line === node.loc.start.line) {
-                        return;
+                        return; // eslint-disable-line no-useless-return
```

Comment:
Sorry, I'm not sure what the correct fix is.
I tried to change the `forEach` to a `for-of` statement, but several tests got failing.